<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stocks</class>
 <widget class="QDialog" name="Stocks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1295</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stocks Detail</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #bfe0f4;</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Drug ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Drug Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Quantity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Company Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_did">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dname">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cmpny">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_qntty">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_sdentry">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color:#2c2255;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Stock entry</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Expiry Date</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_edentry">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color:#2c2255;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Selling Price/per</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_price">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>841</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #bfe0f4;
color:#2c2255;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_entrybtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Drug Entry</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_updbtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Drug Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Drug Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dtype">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Drug Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_drdisplay">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Display Table</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Cost Price/Per</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_costprice">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:rgb(85, 85, 255);
border-bottum: 1px #ffffff</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_search">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#3f76dc;
border-radius:15px;
color:rgb(223, 223, 223);
}
QPushButton:hover{
color:red;
border-radius:15px;
background:#333;

}</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
